setup.py

```diff
@@ -123,7 +123,7 @@ def run(self) -> None:
         "License :: OSI Approved :: Apache Software License",
         "Operating System :: OS Independent",
     ],
-    python_requires=">=3.8",
+    python_requires=">=3.7",
```

Comment:
Does it still require new features introduced in 3.7?

Reply:
I don’t think so, but that’s something I will test. I'll see how far back we can go. There was also a build issue which doesn’t make a lot of sense so investigating that as well.